package.json

```diff
@@ -37,6 +39,7 @@
     "babel-preset-react-app": "^9.1.2",
     "chalk": "^2.4.2",
     "clog-cli": "^1.0.0",
+    "core-js": "^3.0.1",
```

Comment:
Needed to add this due to https://github.com/storybookjs/storybook/blob/master/MIGRATION.md#core-js-dependency-errors

Reply:
Would yarn resolutions work for this instead of the explicit dev-dependency? https://classic.yarnpkg.com/en/docs/selective-version-resolutions

Reply:
[Looks like it should be possible](https://github.com/storybookjs/storybook/issues/11255#issuecomment-673899817), although kind of ugly. 

I thought it probably didn't matter much since it's just a devDependency?